debuginfo/src/dwarf.rs

```diff
@@ -988,6 +999,20 @@ impl<'s> Iterator for DwarfUnitIterator<'s> {
 
 impl std::iter::FusedIterator for DwarfUnitIterator<'_> {}
 
+/// Options to use with functions_option.
+#[derive(Debug)]
+pub struct DwarfFunctionIteratorOption {
+    collapse_lines: bool,
```

Comment:
this should be `pub`, or have some kind of builder pattern, otherwise you have no outside way to set it to false right now, do you?

Reply:
good catch, thx